### ISSUE TITLE
WebP animations don't respect loop count

### ISSUE THREAD

**Issue by eresid** [facebook/fresco#180]
I created animation:
```sh
$ webpmux -frame smile_xhdpi_01.webp +41+0+0+1 \
-frame smile_xhdpi_02.webp +41+0+0+1 \
-frame smile_xhdpi_03.webp +41+0+0+1 \
-frame smile_xhdpi_04.webp +41+0+0+1 \
-frame smile_xhdpi_05.webp +41+0+0+1 \
-loop 2 -bgcolor 0,0,0,0 -o anim_container.webp
```
And how set loop count in code? I not found parameter

**Comment by eresid** [comment]
-loop 1:
https://dl.dropboxusercontent.com/u/6610969/webp_180_example.webp

In Chrome animation plays once, in fresco has not limit

**Comment by tyronen** [comment]
OK I see. I was able to reproduce this too.

**Comment by facebook-github-bot-0** [comment]
Thank you for reporting this issue and appreciate your patience. We've  notified the core team for an update on this issue. We're looking for a response within the next 30 days or the issue may be closed.

**Comment by tyronen** [comment]
Fixed in v0.8.0.

**Comment by haseebrabbani** [comment]
@tyronen does the fix also respect loop counts for GIFs? I'm still seeing my GIF loop infinitely even though I have set the loop count to 1

**Comment by lambdapioneer** [comment]
Hi @haseebrabbani and @nicolasbrugneaux,

The loop count for GIFs should be fixed by https://github.com/facebook/fresco/commit/95359b9cb7650cf00b362ffbc018e90d503d476b which is included in 0.13.0. If you upgrade to the latest release, things should be alright for GIFs.

**Comment by nicolasbrugneaux** [comment]
It works!! Thanks!
